### PR TITLE
Add recipe for pytorch-pretained-bert

### PR DESCRIPTION
**Pull request by CurtLH**

### CODE REVIEW

**Comment by CurtLH** [comment]
This recipe is ready for review @conda-forge/staged-recipes

File: recipes/pytorch-pretrained-bert/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - pytorch_pretrained_bert
+
+about:
+  home: https://pypi.org/project/pytorch-pretrained-bert/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
```
Comment (scopatz) [inline recipes/pytorch-pretrained-bert/meta.yaml]: See https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Comment (sodre) [reply recipes/pytorch-pretrained-bert/meta.yaml]: I think in CB3, conda will look for the license in both places. Is there a particular reason why we chose to use the environment one?

Comment (scopatz) [reply recipes/pytorch-pretrained-bert/meta.yaml]: I don't think that is true... but I could be wrong.

Comment (sodre) [reply recipes/pytorch-pretrained-bert/meta.yaml]: I found the commit where that feature was added:

https://github.com/conda/conda-build/commit/85c46be3bdbd4dc13cc9be0b6db45c1eeac91d11

Is it worth it to update the documentation indicating that either mode is valid?

**Pull request merged**